source/support.html.erb

```diff
@@ -126,7 +126,7 @@ title: Support
           tom.dolan@digital.cabinet-office-gov.uk
         </a>
         <br />
-        Jess O’Leary, product manager -
+        Jess O'Leary, product manager -
```

Comment:
This is a correct apostrophe and shouldn’t be changed (except in the email address where it won’t work) – see http://smartquotesforsmartpeople.com